test_httpbin_ip.py

```diff
@@ -48,3 +51,39 @@ def test_basic_auth_2(base_url, user, password):
     r = get(base_url + 'basic-auth/{0}/{1}'.format(user, password), auth=(user, password))
     assert r.status_code == 200
     assert r.json() == {'authenticated': True, 'user': user}
+
+
+def test_deflated(base_url, my_ip):
+    r = get(base_url + 'deflate')
+    assert r.json()['method'] == 'GET'
+    assert r.json()['origin'] == my_ip
+    assert r.status_code == requests.codes.ok
+    assert r.json()['headers']['Host'] == base_url[7:-1]
+    assert r.json()['headers']['Accept'] == '*/*'
+    assert r.json()['headers']['Accept-Encoding'] == 'gzip, deflate'
+    if base_url == 'http://localhost:8000/':
+        assert r.json()['headers']['Connection'] == 'keep-alive'
+    elif base_url == 'http://httpbin.org/':
+        assert r.json()['headers']['Connection'] == 'close'
```

Comment:
connection i think depends on the request headers. it doesn't matter which environment you send the request to:
```
# here i didn't specify  connection in the request and it came back as 'keep-alive' by default from the prod env
>>> r = get('http://httpbin.org/deflate')
>>> r.headers['connection']
'keep-alive'
>>> r.request.headers['connection']
'keep-alive'

# here i specify 'close' in my request headers and it comes back 'close'
>>> get('http://httpbin.org/deflate', headers={'Connection': 'close'})
<Response [200]>
>>> r = get('http://httpbin.org/deflate', headers={'Connection': 'close'})
>>> r.headers['connection']
'close'
>>> r.request.headers['connection']
'close'
```

I think you get 'close' my default in travisci because the request goes through proxy... but i may be wrong.
More info on this here:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Connection
and for super botans:
https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html